src/Sylius/Component/Attribute/composer.json

```diff
@@ -25,7 +25,8 @@
         "sylius/resource": "0.17.*",
         "sylius/translation": "0.17.*",
         "sylius/registry": "0.17.*",
-        "symfony/validator": "^2.7.7"
+        "symfony/validator": "^2.7.7",
+        "doctrine/collections": "~1.2"
```

Comment:
I got that value from here:

https://github.com/Sylius/Sylius/blob/master/composer.json#L28